### ISSUE TITLE
Impossible d'installer Zord

### ISSUE THREAD

**Issue by hantsaniala** [DrozNumerique/Zord#1]
Bonjour,

D'abord, j'aimerai demander si c'est le même plateforme que http://flexedo.ivvq.droz.org.

Par la suite, lors de l'installation, je rencontre ce genre de problème. Pouvez-vous m'aider ? Merci.

![Opera Snapshot_2019-07-07_165547_localhost](https://user-images.githubusercontent.com/8157490/60785968-83655f00-a15d-11e9-97ed-d7cc47ce807d.png)

**Comment by eark3** [comment]
Oui, il s'agit bien de la même plateforme que http://flexedo.ivvq.droz.org
Mais il y a un certain nombre de pré-requis à l'installation de Zord (MySQL, SolR, modules PHP...) qui ne sont pas documentés pour l'instant
